pipelines/template-build/template-build.yaml

```diff
@@ -265,7 +265,21 @@ spec:
       workspaces:
       - name: workspace
         workspace: workspace
-
+    - name: rpms-signature-scan
```

Comment:
Do we really need this task for all pipelines and all users?

Reply:
I believe so. It's how Konflux will be able to enforce that all included RPMs are signed with the right key via EC.

Reply:
Yes, @mmorhun thats the feature.
This is the [refinement doc](https://docs.google.com/document/d/1fhwCyXKU8kB5cX4UkxAJ51lEbF_vXG7APTICusU8AsU/edit)
This is the [feature](https://issues.redhat.com/browse/KONFLUX-3149)

cc @ralphbean @lcarva